Assets/MRTK/SDK/Features/UX/Prefabs/RiggedHandVisualizer/RiggedHandLeft bones.prefab.meta

```diff
@@ -1,6 +1,6 @@
 fileFormatVersion: 2
-guid: 89306ab5ab6988249a8aa53c429567a2
-TextScriptImporter:
+guid: 3cfb75756a9f6474481612e89b7931b5
```

Comment:
I don't see the corresponding prefab for this meta in this PR. Should it be added or this removed?

Reply:
need to refactor out a bunch of stuff that popped up from the version diffs, processing now